sentry_dart_plugin/README.md

```diff
@@ -0,0 +1,57 @@
+# Sentry Dart Plugin
+
+A Dart Build Plugin that uploads symbols for Android, iOS/macOS and Web to Sentry via [sentry-cli](https://docs.sentry.io/product/cli/).
+
+For doing it manually, please follow our [docs](https://docs.sentry.io/platforms/flutter/upload-debug/).
+
+## :clipboard: Install
+
+In your `pubspec.yaml`, add `sentry_dart_plugin` as a new dev dependency.
+
+```yaml
+dev_dependencies:
+  sentry_dart_plugin: ^1.0.0-alpha.1
+```
+
+## Run
+
+```bash
+dart run sentry_dart_plugin
+```
+
+The `flutter build apk`, `flutter build ios` (or _macos_) or `flutter build web` is required before executing the `sentry_dart_plugin` plugin.
+
+## Configuration (Optional)
+
+This tool comes with default configuration, you can configure it to suit your needs.
+
+Add `sentry_plugin:` configuration at the end of your `pubspec.yaml` file:
+
+```yaml
+sentry_plugin:
```

Comment:
The package is named `sentry_dart_plugin`. Maybe the yaml key should be the same?

Reply:
or maybe just `sentry`